pointerevents/pointerevent_element_haspointercapture-manual.html

```diff
@@ -56,6 +56,12 @@
                         assert_equals(target0.hasPointerCapture(e.pointerId), true,
                                       "after target0.setPointerCapture, target0.hasPointerCapture should be true");
                     });
+                    // If the element.hasPointerCapture is false element.releasePointerCapture does nothing
+                    target1.releasePointerCapture(e.pointerId);
+                    test_pointerEvent.step(function () {
+                        assert_equals(target0.hasPointerCapture(e.pointerId), true,
+                                      "while target1.hasPointerCapture is false, target1.releasePointerCapture should not change hasPointerCapture");
+                    });
```

Comment:
LGTM.

Nit: spotted a slip from past, please fix: Line 37 should add "_After_ target1.setPointerCapture".